guides/asset_management.md

```diff
@@ -77,6 +89,14 @@ args: ~w(js/app.js --bundle --target=es2022 --outdir=../priv/static/assets/js --
 
 If you need to reference other directories, you need to update the arguments above accordingly. Note running `mix phx.digest` will create digested files for all of the assets in `priv/static`, so your images and fonts are still cache-busted.
 
+### Ensuring fonts and images from third-party libraries are loaded
+If you import a Node package that depends on additional fonts or images, you might find them to fail to load. This is because they are referenced in the JS or CSS but by default Esbuild will not touch or process referenced files. You can add arguments to esbuild in `config/config.exs` to ensure that the referenced resources are copied to the output folder. The following example would copy all referenced font files to the output folder and prefix the paths with `/assets/`:
+
+```elixir
+args: ~w(js/app.js --bundle --target=es2017 --outdir=../priv/static/assets --external:/fonts/* --external:/images/* --public-path=/assets/ --loader:.woff=copy  --loader:.ttf=copy --loader:.eot=copy --loader:.woff2=copy),
+```
+
```

Comment:
For more information, see [the esbuild documentation](https://esbuild.github.io/content-types/#copy).